bravado/testing/integration_test.py

```diff
@@ -1,54 +1,256 @@
+                'responses': {
+                    '200': {
+                        'description': 'HTTP/200',
+                        'schema': {
+                            'type': 'object',
+                            'properties': {
+                                'message': {
+                                    'type': 'string',
+                                },
+                            },
+                            'required': ['message'],
+                        },
+                    },
+                },
+            },
+        },
+        '/char_test/{special}': {
+            'get': {
+                'operationId': 'get_char_test',
+                'produces': ['application/json'],
+                'parameters': [
+                    {
+                        'in': 'path',
+                        'name': 'special',
+                        'type': 'string',
+                        'required': True,
+                    }
+                ],
+                'responses': {
+                    '200': {
+                        'description': 'HTTP/200',
+                        'schema': {'$ref': '#/definitions/api_response'},
+                    },
+                },
+            },
+        },
+        '/sanitize-test': {
+            'get': {
+                'operationId': 'get_sanitized_param',
+                'produces': ['application/json'],
+                'parameters': [
+                    {
+                        'in': 'header',
+                        'name': 'X-User-Id',
+                        'type': 'string',
+                        'required': True,
+                    }
+                ],
+                'responses': {
+                    '200': {
+                        'description': 'HTTP/200',
+                        'schema': {'$ref': '#/definitions/api_response'},
+                    },
+                },
+            },
+        },
+        '/sleep': {
+            'get': {
+                'operationId': 'sleep',
+                'produces': ['application/json'],
+                'parameters': [
+                    {
+                        'in': 'query',
+                        'name': 'sec',
+                        'type': 'number',
+                        'format': 'float',
+                        'required': True,
+                    }
+                ],
+                'responses': {
+                    '200': {
+                        'description': 'HTTP/200',
+                        'schema': {'$ref': '#/definitions/api_response'},
+                    },
+                },
+            },
+        },
+    },
+}
 
-class ServerClientGeneric:
-    """
-    Generic class to run integration tests with the different HTTP clients definitions
-    """
 
-    http_client_type = None
-    http_future_adapter_type = None
-    connection_errors_exceptions = None
+@bottle.get('/swagger.json')
+def swagger_spec():
+    return SWAGGER_SPEC_DICT
```

Comment:
As discussed offline: if bottle doesn't set the content type automatically then let's add it here too.

Reply:
Actually setting content_type to application/json is not needed as bottle does that by default.
I'm going to remove them